src/core/Card/index.js

```diff
@@ -23,7 +23,11 @@ const useStyles = makeStyles(theme => ({
     backgroundColor: 'whitesmoke',
     width: expand ? '100%' : width || 500,
     height: 'auto',
-    padding: 20
+    padding: 20,
+    '& > img': {
```

Comment:
This needs to be `& img`, otherwise it will only be applicable to direct children and not an image, say, in a paragraph:

![Screenshot from 2020-01-23 17-03-22](https://user-images.githubusercontent.com/1779590/72991141-7448b480-3e02-11ea-80ef-7af03b9ca850.png)